src/main/java/com/mg105/user_interface/inventory/InventoryDisplay.java

```diff
@@ -0,0 +1,163 @@
+package com.mg105.user_interface.inventory;
+
+import com.mg105.interface_adapters.inventory.InventoryController;
+import com.mg105.user_interface.AlertBox;
+import com.mg105.utils.PartyConstants;
+import javafx.scene.Scene;
+import javafx.scene.control.Button;
+import javafx.scene.control.ComboBox;
+import javafx.scene.control.Label;
+import javafx.scene.layout.HBox;
+import javafx.scene.layout.VBox;
+import javafx.scene.paint.Color;
+import javafx.stage.Modality;
+import javafx.stage.Stage;
+import org.jetbrains.annotations.NotNull;
+
+import java.util.HashMap;
+
+/**
+ * A user interface class that displays the state of the inventory to the user and allows them to
+ * perform actions that relate to the inventory and its items.
+ */
+
+public class InventoryDisplay implements InventoryViewInterface {
+    private VBox itemsDisplay;
+    private final HashMap<String, HBox> itemNameToInfo = new HashMap<>();
+    private final Stage window = new Stage();
+    private InventoryController controller;
+    private String characterSelected = PartyConstants.ALL_PARTY_MEMBER_NAMES[0];
+
+    public InventoryDisplay(InventoryController controller) {
+        this.controller = controller;
+    }
+
+    private String getCharacterSelected() {
```

Comment:
Since this is a private method, why not just write this.characterSelected wherever you call it instead?